gradlew

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```

Comment:
Please remove the gradle wrapper. I don't want to encourage that concept, because it introduces an additional attack surface. [For example, it required @dschuermann to introduce gradlew verification](https://docs.gradle.org/2.6-rc-1/release-notes.html).